helmfile.d/charts/networkpolicy/service-cluster/templates/ingress-nginx/controller.yaml

```
@@ -12,9 +12,24 @@ spec:
     - Ingress
     - Egress
   ingress:
-    {{- if or  .Values.global.scNodes.ips .Values.global.scIngress.ips }}
+    {{- if eq (dig "ck8sCloudProvider" "" .Values.global ) "azure" }}
     - from:
-      {{- if not (or .Values.global.externalLoadBalancer .Values.global.ingressUsingHostNetwork) }}
+      - ipBlock:
+          cidr: 0.0.0.0/0
+      ports:
+        - protocol: TCP
+          port: 443
+        - protocol: TCP
+          port: 80
+    {{- end }}
+    {{- if or .Values.ingressNginx.ingressOverride.ips .Values.global.scNodes.ips .Values.global.scIngress.ips }}
```

Question: Shouldn't this use the `enabled` config instead of the ips?

```suggestion
    {{- if or .Values.ingressNginx.ingressOverride.enabled .Values.global.scNodes.ips .Values.global.scIngress.ips }}
```

---

I copied what we had before [removing the config option](https://github.com/elastisys/compliantkubernetes-apps/commit/52be5bb899e652ce3bed33131839b608f6042497#diff-0322ba789b1de215383aead5b31fb72c7de2f36536b72c39b5d9bc59fd11acf6L15), but I think it kinda makes sense as it will throw an error if you have enabled it but not configured any IPs

---

Maybe checking both would be more correct

---

Right, but lets keep it as it was for this.